tests/test_post_to_studio.py

```diff
@@ -370,3 +383,15 @@ def test_post_to_studio_name(tmp_dir, mocked_dvc_repo, mocked_studio_post):
         "https://0.0.0.0/api/live",
         **get_studio_call("start", exp_name="custom-name"),
     )
+
+
+def test_post_to_studio_if_done_skipped(tmp_dir, mocked_dvc_repo, mocked_studio_post):
+    live = Live()
+    live._studio_events_to_skip.add("start")
+    live._studio_events_to_skip.add("done")
+    live.log_metric("foo", 1)
+    live.end()
+
+    mocked_post, _ = mocked_studio_post
+    call_types = [call.kwargs["json"]["type"] for call in mocked_post.call_args_list]
+    assert "data" in call_types
```

Comment:
Added test to check the use case from [slack](https://iterativeai.slack.com/archives/CUSNDR35K/p1697630212171659), where `live.next_step()` is never called and `done` events are skipped.